{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/site/models.py

```diff
@@ -1,10 +1,116 @@
 from cms.apps.pages.models import Page
+from django.contrib.auth.models import (AbstractBaseUser, BaseUserManager,
+                                        PermissionsMixin)
 from django.db import models
 from django.utils.functional import cached_property
+from django.utils.timezone import now
 
 from ...utils.models import HasLinkMixin
 
 
+class UserManager(BaseUserManager):
```

Comment:
I would really like this to be broken out into a separate PR, and definitely not in the `site` app.